From 13bbefd57e8dcabae650f4a02e667d5c003c289f Mon Sep 17 00:00:00 2001
From: Diego Biurrun <diego@biurrun.de>
Date: Mon, 22 Oct 2012 16:16:07 +0200
Subject: [PATCH 035/278] doxygen: Add av_alloc_size to list of predefined
 macros

This avoids Doxygen believing the attribute is the function name.
---
 doc/Doxyfile |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/doc/Doxyfile b/doc/Doxyfile
index eb394f0..aa1f4e2 100644
--- a/doc/Doxyfile
+++ b/doc/Doxyfile
@@ -1375,7 +1375,8 @@ INCLUDE_FILE_PATTERNS  =
 
 PREDEFINED             = "__attribute__(x)=" \
                          "DECLARE_ALIGNED(a,t,n)=t n" \
-                         "offsetof(x,y)=0x42"
+                         "offsetof(x,y)=0x42" \
+                         av_alloc_size \
 
 # If the MACRO_EXPANSION and EXPAND_ONLY_PREDEF tags are set to YES then
 # this tag can be used to specify a list of macro names that should be expanded.
-- 
1.7.9.5

